nuxt-frontend/components/flutter-app-development/CtaSection.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <section>
+    <div class="relative h-[40vh] lg:h-[65vh] 2xl:h-[670px]">
+      <img
+        :src="background400"
+        :srcset="`${background400} 400w, ${background800} 800w`"
+        class="h-full w-full object-cover opacity-20 lg:hidden"
+        alt="flutter-cta"
+        loading="lazy"
+      />
+      <img
+        :src="background1200"
+        :srcset="`${background1200} 1200w, ${background2400} 2400w`"
```

Comment:
i think in this background images for both mobile and desktop we should use blur images given in figma